tests/functional/builtins/codegen/test_abi_decode.py

```diff
@@ -500,15 +505,18 @@ def f(x: Bytes[32 * 3]):
     """
     c = get_contract(code)
 
-    data = method_id("f(bytes)")
-    payload = (
-        0x20,  # tuple head
-        0x60,  # parent array length
+    msg_call_overhead = (method_id("f(bytes)"), 0x20, 0x60)  # tuple head  # parent array length
+
+    data = _abi_payload_from_tuple(msg_call_overhead, MSG_CALL_ABI_OVERHEAD_SZ)
+
+    buffer_payload = (
         # parent payload - this word will be considered as the head of the abi-encoded inner array
         # and it will be added to base ptr leading to an arithmetic overflow
-        2**256 - 0x60,
+        2**256
+        - 0x60,
```

Comment:
weird formatting

Reply:
result of linting

Reply:
can extract the comment to the previous line, then this line will just be `buffer_payload = (2**256 - 0x60,)`

Reply:
https://github.com/vyperlang/vyper/pull/4096/commits/576efa3c23a2d6be40f7875495c3aeb39875f958